### ISSUE TITLE
Skip prompting for Dockerfile if Docker Engine is not running

### ISSUE THREAD

**Issue by efekarakus** [aws/copilot-cli#1814]
## Background

Currently Copilot prompts for which Dockerfile to use while running `copilot svc init` or `copilot job init`:
```
  Which *Dockerfile* would you like to use for my-svc?
  > ./Dockerfile
     Enter a custom path for your Dockerfile
     Use an existing image instead
```

## Ask

Instead, it'd be nice if Copilot can detect if Docker Engine is running or not and filter the options accordingly, for example we can log a statement and immediately jump to using an existing image instead:
```
Docker Engine is not running, Copilot won't build from a Dockerfile: docker: command not found

   What's the location of the image to use? nginx
```

## Proposed programming model
One possible way to tell if the daemon is running or not is by running `docker info -f '{{json . }}'` and printing the error that we receive. For example, if docker is not installed:
```
$ docker info -f '{{json . }}'
bash: docker: command not found
```
Or if the daemon is not running:
```
$ docker info -f '{{json . }}' 
{
  "ServerErrors": [
    "Error response from daemon: dial unix docker.raw.sock: connect: connection refused"
  ],
  ...
}
```

**Comment by iamhopaul123** [comment]
closed via #1821, #1830, #1836

**Comment by efekarakus** [comment]
This was released in https://github.com/aws/copilot-cli/releases/tag/v1.1.0!
